fixtures/eurolite/led-kls-801.json

```diff
@@ -0,0 +1,312 @@
+          }
+        },
+        {
+          "range": [250, 255],
+          "name": "Sound control",
+          "switchChannels": {
+            "Red 1 / Running Speed": "Running Speed",
+            "Green 1 / Fade Time": "Fade Time"
+          }
+        }
+      ]
+    },
+    "Master Dimmer": {
+      "type": "Intensity",
+      "highlightValue": 255,
+      "crossfade": true
+    },
+    "Strobe": {
+      "type": "Strobe",
+      "highlightValue": 220,
+      "capabilities": [
+        {
+          "range": [0, 9],
+          "name": "Strobe off"
+        },
+        {
+          "range": [10, 255],
+          "name": "Strobe slow-fast",
+          "menuClick": "center"
+        }
+      ]
+    },
+    "Red 1": {
+      "type": "SingleColor",
+      "color": "Red",
+      "highlightValue": 255,
+      "crossfade": true
+    },
+    "Running Speed": {
+      "type": "Speed",
+      "crossfade": true,
+      "capabilities": [
+        {
+          "range": [0, 255],
+          "name": "slow-fast"
+        }
+      ]
+    },
+    "Green 1": {
+      "type": "SingleColor",
+      "color": "Green",
+      "highlightValue": 255,
+      "crossfade": true
+    },
+    "Fade Time": {
+      "type": "Speed",
+      "crossfade": true,
+      "capabilities": [
+        {
+          "range": [0, 255],
+          "name": "slow-fast"
+        }
+      ]
+    },
+    "Blue 1": {
+      "type": "SingleColor",
+      "color": "Blue",
+      "highlightValue": 255,
+      "crossfade": true
+    },
+    "Red 2": {
+      "type": "SingleColor",
+      "color": "Red",
+      "highlightValue": 255,
+      "crossfade": true
+    },
+    "Green 2": {
+      "type": "SingleColor",
+      "color": "Green",
+      "highlightValue": 255,
+      "crossfade": true
+    },
+    "Blue 2": {
+      "type": "SingleColor",
+      "color": "Blue",
+      "highlightValue": 255,
+      "crossfade": true
+    },
+    "Red 3": {
+      "type": "SingleColor",
+      "color": "Red",
+      "highlightValue": 255,
+      "crossfade": true
+    },
+    "Green 3": {
+      "type": "SingleColor",
+      "color": "Green",
+      "highlightValue": 255,
+      "crossfade": true
+    },
+    "Blue 3": {
+      "type": "SingleColor",
+      "color": "Blue",
+      "highlightValue": 255,
+      "crossfade": true
+    },
+    "Red 4": {
+      "type": "SingleColor",
+      "color": "Red",
+      "highlightValue": 255,
+      "crossfade": true
+    },
+    "Green 4": {
+      "type": "SingleColor",
+      "color": "Green",
+      "highlightValue": 255,
+      "crossfade": true
+    },
+    "Blue 4": {
+      "type": "SingleColor",
+      "color": "Blue",
+      "highlightValue": 255,
+      "crossfade": true
+    }
+  },
+  "heads": {
+    "LED1": ["Red 1", "Green 1", "Blue 1"],
+    "LED2": ["Red 2", "Green 2", "Blue 2"],
+    "LED3": ["Red 3", "Green 3", "Blue 3"],
+    "LED4": ["Red 4", "Green 4", "Blue 4"]
+  },
+  "modes": [
+    {
+      "name": "Control 3-channel",
```

Comment:
Please remove all modes except *Combined 15-channel* (renamed to *15-channel*) since the other ones were just a workaround as we had no way of defining switching channels.